cpp/models/abm/person.h

```diff
@@ -68,6 +69,15 @@ class Person
         return (m_person_id == other.m_person_id);
     }
 
+    /**
+     * @brief The TestResult of a Person.
+     */
+    struct TestResult {
```

Comment:
To keep this separate, I would put this in test_type.h.

Reply:
Yes, it is updated!